Documentation/Contributing/development-flow.md

```diff
@@ -243,6 +248,27 @@ Common cases that may need tests:
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
 
+## Linting
+
+The CI will run various linters against your changes to check code, manifests, and docs for syntactic correctness.
+The various linters can be run selectively locally as follows.
+
+Run `make lint` to run all linters. This will require the following CLI tools to be installed on the system:
+
+1. [yamllint](https://www.yamllint.com/)
+2. [shellcheck](https://www.shellcheck.net/)
+3. [pylint](https://www.pylint.org/)
+4. [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2)
+5. [go vet](https://pkg.go.dev/cmd/vet) - part of the golang installation.
+6. [golangci-lint](https://golangci-lint.run/)
+
```

Comment:
remove extra line

Reply:
removing it lets markdownlint fail: lists need to be surrounded by blank lines.